### ISSUE TITLE
Maybe make any project work automatically.

### ISSUE THREAD

**Issue by howardroark** [arshaw/ghi-scoreboard#5]
Hey!

I really like the approach with labels.  The one thing I'm right away curious to do is to look at other projects or my own.  Maybe the app could just sync github's url structure so that `ghi-scoreboard.com/user/project` just works.

**Comment by arshaw** [comment]
@howardroark that's a really good idea. it'd be easy to set up very simplistic scoreboards like http://arshaw.com/moment-scoreboard/ , but anything that requires [aggregation](https://github.com/arshaw/ghi-scoreboard#aggregating-data) would need to be self-hosted. but still very worthwhile and would serve as a great demo!

**Comment by howardroark** [comment]
Yeah, that part is tricky.  But I'm sure a single VM could manage it ;)

Was playing around with a concept like this as well... https://github.com/coco/gitsup Didn't get too far past the initial proof of concept ;) It's a meteor js app running for free on heroku. It basically stores an issue if it had a +/- reaction. As a sort of byproduct of how meteor works they sort of bubble in as they are found via the API.

I think what you have made here is very well considered in terms of usability :)   Though it is kinda nice to be able to plug in any source and (sorta) quickly get a ranked list of issues (or whatever else).

**Comment by arshaw** [comment]
gitsup looks really cool!

yeah, maybe i'll set up a little app to emulate the simple non-aggregation scenario.

aggregation is hard because it requires a lot of calls to github's API. if i allowed *anybody* to do it via my app, i'd quickly hit the request limit.

PS- i want to consider to start considering thumbs-downs, like you are. issue: https://github.com/arshaw/ghi-scoreboard/issues/4
